### PR TITLE
TELCODOCS-261: Second round of What's new updates for Redfish HW events

### PR DESCRIPTION
**Pull request by tmulquee**
Jira ticket: https://issues.redhat.com/browse/TELCODOCS-261

For 4.10

https://deploy-preview-39520--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-networking

Note: moved location to Networking after discussion with @jzding .

https://deploy-preview-39520--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-technology-preview

### CODE REVIEW

**Comment by tmulquee** [comment]
@jzding @nwaizer Please review the text in the preview link above. Can you let me know if this story is for Technical Preview? I have added a row for this feature in the Technical Preview table in the Rel note.

**Comment by tmulquee** [comment]
@jzding @nwaizer Please review the text in the above RN link. Also check the entry in the Technology Preview table - is this feature TP or GA.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -63,6 +63,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-10-networking"]
 === Networking
 
+[id="ocp-4-10-hardware"]
+=== Hardware
+
+[id="ocp-4-10-redfish-events"]
+==== Low-latency Redfish hardware event delivery (Technology Preview)
+In a bare metal cloud environment, applications must act on hardware changes and failures quickly with high reliability. OpenShift Container Platform now provides a hardware event proxy that enables such applications to respond to Redfish hardware events with low latency.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:71]: Is "bare metal cloud" an oxymoron? Also, "bare metal" in the adjectival postion takes a hyphen.
 
```suggestion
{product-title} now provides a hardware event proxy that enables applications running on bare-metal clusters to respond quickly to Redfish hardware events such as hardware changes and failures.
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -63,6 +63,18 @@ This release adds improvements related to the following components and concepts.
...
+
+The hardware event proxy supports a publish-and-subscribe service through which hardware events detected by Redfish can be consumed by relevant applications. The proxy must be running on hardware that supports Redfish v1.8 and higher.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:73]: publish-subscribe (pub/sub)

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -63,6 +63,18 @@ This release adds improvements related to the following components and concepts.
...
+
+The applications can be developed using a REST API to consume and respond to events such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure. Reliable end-to-end messaging without persistent stores is based on the Advanced Message Queuing Protocol (AMQP). The latency of the messaging service is in the 10 millisecond range.
+Note: This feature is only supported in this release in the context of Single Node OpenShift.
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:76]: Use asciidoc syntax for notes. 

```
[NOTE]
====
This feature is only supported for single node OpenShift clusters.
====
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -552,6 +564,11 @@ In the table below, features are marked with the following statuses:
 |TP
 |
 
+| Low-latency Redfish hardware event delivery
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:567]:
```suggestion
| Low-latency Redfish hardware events
```

**Review by aireilly — COMMENTED** [review]
A few comments for you Tony :)

File: release_notes/ocp-4-10-release-notes.adoc (hunk 4 above)
Comment (tmulquee) [reply release_notes/ocp-4-10-release-notes.adoc:567]: Thanks for the edits, Aidan.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 5]
```diff
@@ -19,7 +19,11 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 {product-title} {product-version} clusters are available at https://cloud.redhat.com/openshift. The {cloud-redhat-com} application for {product-title} allows you to deploy OpenShift clusters to either on-premise or cloud environments.
 
 // Double check OP system versions
+<<<<<<< HEAD
 {product-title} {product-version} is supported on {op-system-base-full} 8.4, as well as on {op-system-first} 4.10.
+=======
+{product-title} {product-version} is supported on {op-system-base-full} 7.9 and 8.4, as well as on {op-system-first} 4.9.
+>>>>>>> 7bfffb698... TELCODOCS-261: Redfish hardware events
```
Comment (ctauchen) [inline release_notes/ocp-4-10-release-notes.adoc:26]: @tmulquee Looks like you've got a bit of merge conflict residue.

Comment (tmulquee) [reply release_notes/ocp-4-10-release-notes.adoc]: Hi Chris, thanks, I believe that should be clean now.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 6]
```diff
@@ -138,6 +138,22 @@ A new `--node-selector` argument provides a way to identify which nodes you are
 
 For more information, see xref:../support/gathering-cluster-data.adoc#support-network-trace-methods_gathering-cluster-data[Network trace methods] and xref:../support/gathering-cluster-data.adoc#support-collecting-host-network-trace_gathering-cluster-data[Collecting a host network trace].
 
+[id="ocp-4-10-hardware"]
+=== Hardware
+
+[id="ocp-4-10-redfish-events"]
+==== Low-latency Redfish hardware event delivery (Technology Preview)
+{product-title} now provides a hardware event proxy that enables applications running on bare-metal clusters to respond quickly to Redfish hardware events such as hardware changes and failures.
```
Comment (lpettyjo) [inline release_notes/ocp-4-10-release-notes.adoc:146]: Add comma: ...hardware events, such as ....

File: release_notes/ocp-4-10-release-notes.adoc [hunk 7]
```diff
@@ -138,6 +138,22 @@ A new `--node-selector` argument provides a way to identify which nodes you are
...
+
+The hardware event proxy supports a publish-subscribe service through which hardware events detected by Redfish can be consumed by relevant applications. The proxy must be running on hardware that supports Redfish v1.8 and higher.
```
Comment (lpettyjo) [inline release_notes/ocp-4-10-release-notes.adoc:148]: Can this language be simplified to: ...publish-subscribe service that allows relevant applications to consume hardware events detected by Redfish.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 8]
```diff
@@ -138,6 +138,22 @@ A new `--node-selector` argument provides a way to identify which nodes you are
...
+
+The applications can be developed using a REST API to consume and respond to events such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure. Reliable end-to-end messaging without persistent stores is based on the Advanced Message Queuing Protocol (AMQP). The latency of the messaging service is in the 10 millisecond range.
```
Comment (lpettyjo) [inline release_notes/ocp-4-10-release-notes.adoc:150]: Change to active voice and add comma: You can use a REST API to develop applications to consume and respond to events, such as...

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM

**Comment by tmulquee** [comment]
@lpettyjo Thanks for the edits, I've put them through.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 9]
```diff
@@ -114,6 +114,27 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+[id="ocp-4-10-redfish-events"]
+==== Low-latency Redfish hardware event delivery (Technology Preview)
+
+{product-title} now provides a hardware event proxy that enables applications running on bare-metal clusters to respond quickly to Redfish hardware events, such as hardware changes and failures.
+
+The hardware event proxy supports a publish-subscribe service that allows relevant applications to consume hardware events detected by Redfish. The proxy must be running on hardware that supports Redfish v1.8 and higher.
+
+You can use a REST API to develop applications to consume and respond to events such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure. Reliable end-to-end messaging without persistent stores is based on the Advanced Message Queuing Protocol (AMQP). The latency of the messaging service is in the 10 millisecond range.
+
+An Operator is provided to manage the lifecycle of the `hw-event-proxy` container:
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:126]: Suggest explicitly naming the Operator and saying where you can install it from.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 10]
```diff
@@ -114,6 +114,27 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
...
+
+* creating
```
Comment (aireilly) [inline release_notes/ocp-4-10-release-notes.adoc:128]: I don't think these lines are required. It's implicit in being deployed as an Operator that the lifecycle of the functionality is handled automatically.

**Review by aireilly — COMMENTED** [review]
Looks good Tony, left a few small comments.

File: release_notes/ocp-4-10-release-notes.adoc (hunk 10 above)
Comment (tmulquee) [reply release_notes/ocp-4-10-release-notes.adoc]: Thanks for the edits Aidan, implemented.

**Pull request merged**